src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx

```diff
@@ -255,4 +255,31 @@
   <data name="FixNumberedComments" xml:space="preserve">
     <value>Fix numbered comments</value>
   </data>
+  <data name="DoNotCallGetTestAccessorDescription" xml:space="preserve">
+    <value>GetTestAccessor() is a helper method reserved for testing. Production code must not call this member.</value>
```

Comment:
I think that you might want to put single quotes around `GetTestAccessor`.

Reply:
GetTestAccessor is a symbol name, but it is not a placeholder. It was my understanding that the single quotes were used for placeholders in the string.

Reply:
The compiler isn't consistent here. Sometimes it includes the quotes and other times not.

Reply:
Yeah I have seen that the messages are inconsistent. In #3183 following some suggestions from @mavasani we have started to try and put single quotes also around symbol names but maybe I didn't get it right. Anyway feel free to disregard my comment.

Reply:
I'll update this later according to the results of #3183 and an internal review with multiple teams.